cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -18,28 +18,16 @@ package options
 
 import (
 	"errors"
-	"fmt"
 	"io/ioutil"
 	"os"
 
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/apimachinery/pkg/runtime/serializer/json"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/kubernetes/pkg/apis/componentconfig"
+	componentconfigscheme "k8s.io/kubernetes/pkg/apis/componentconfig/scheme"
 	componentconfigv1alpha1 "k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1"
 )
 
-var (
-	configScheme = runtime.NewScheme()
```

Comment:
why remove this local scheme? This was the better approach.

Reply:
See the other comment